package.json

```diff
@@ -19,10 +19,12 @@
   },
   "dependencies": {
     "@observablehq/plot": "^0.2.0",
+    "autoprefixer": "^10.3.4",
     "d3-array": "^3.0.2",
     "d3-format": "^3.0.1",
     "d3-time": "^3.0.0",
     "d3-time-format": "^4.0.0",
+    "postcss-cli": "^8.3.1",
```

Comment:
Do we need this?

Reply:
probably not? I must have copy-pasted from one README or another... will remove